recipes/cms-fireworks/meta.yaml

```diff
@@ -31,62 +28,52 @@ requirements:
     - {{ cdt('libselinux') }}  # [linux]
     - {{ cdt('libxdamage') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
-
   host:
-    - {{ compiler('c') }}  # [linux]
-    - {{ compiler('cxx') }}  # [linux]
-    - {{ compiler('fortran') }}  # [linux]
+    - boost =1.68.0
+    - clhep =2.4.0.2
+    - cms-md5
+    - hepmc2
+    - libsigcpp
+    - libuuid
+    # - openssl
+    - pybind11
+    - python
+    - root
+    - tbb
+    - tinyxml2
+    - xerces-c
     - xorg-libx11  # [linux]
     - xorg-libxau  # [linux]
     - xorg-libxcursor  # [linux]
     - xorg-libxext  # [linux]
     - xorg-libxfixes  # [linux]
     - xorg-libxft  # [linux]
     - xorg-libxpm  # [linux]
-    - libgcc-ng >=7.3.0  # [linux]
-    - libstdcxx-ng >=7.3.0  # [linux]
-    - libcxx >=4.0.1  # [osx]
-    - libuuid  # [osx]
-    - python =2.7
-    - root
-    - boost-cpp =1.68.0 
-    - boost =1.68.0 
-    - libsigcpp
-    - clhep =2.4.0.2
-    - xerces-c
-    - pybind11
+    - xz
+    - zlib
+  run:
+    - {{ pin_compatible('boost', max_pin='x.x') }}
+    - {{ pin_compatible('clhep', max_pin='x.x.x.x') }}
+    - cms-md5
     - hepmc2
+    - libsigcpp
+    # - openssl
+    - python
+    - tbb
     - tinyxml2
-    - cms-md5
-  run:
     - xorg-libx11  # [linux]
     - xorg-libxau  # [linux]
     - xorg-libxcursor  # [linux]
     - xorg-libxext  # [linux]
     - xorg-libxft  # [linux]
     - xorg-libxpm  # [linux]
-    - libgcc-ng >=7.3.0  # [linux]
```

Comment:
I did this to get the version of boost-cpp compiled with std=c++14. This is needed for the cmsRun* executables. Otherwise there is an error linking libboost_program_options.